package.json

```diff
@@ -53,6 +54,7 @@
     "glob": "^7.1.4",
     "grammkit": "0.7.0",
     "http-server": "^0.11.1",
+    "js-green-licenses": "^0.5.0",
```

Comment:
We should discuss this with team in chat before adding a dependency. Can you raise this there, link the PR, and explain what using this brings us vs. if we do without?

Reply:
Ack, will do.  This is published in google's github repo despite it being an "unofficial" project.

This is the tool that inspects the transitive closure of the npm installed dependencies and verifies that all licenses are acceptable.